docs/sampleConfigurations/DscResources/Microsoft.WindowsSandbox.DSC/full.sandbox.dsc.yaml

```diff
@@ -32,12 +29,14 @@ properties:
         HostFolder: C:\Sandbox
         SandboxFolder: C:\Sandbox
         LogonCommand: >
-          cmd /c start powershell -NoExit -Command "Write-Host 'Installing NuGet package provider...';
-          Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force;
-          Write-Host `n'Installing Microsoft.WinGet.Client PowerShell module...' `n;
-          Install-Module Microsoft.WinGet.Client -RequiredVersion 0.2.1 -Repository PSGallery -Force;
+          cmd /c start powershell -NoExit -Command "$progressPreference = 'silentlyContinue';
           Write-Host 'Setting execution policy to remote signed...' `n;
           Set-ExecutionPolicy RemoteSigned -Force;
-          Write-Host 'Running the Repair-WinGetPackageManager cmdlet to install App Installer (WinGet)...' `n;
-          Repair-WingetPackageManager
+          Write-Host 'Downloading WinGet and its dependencies...' `n;
+          Invoke-WebRequest -Uri https://aka.ms/getwinget -OutFile Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle;
+          Invoke-WebRequest -Uri https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx -OutFile Microsoft.VCLibs.x64.14.00.Desktop.appx;
+          Invoke-WebRequest -Uri https://github.com/microsoft/microsoft-ui-xaml/releases/download/v2.8.6/Microsoft.UI.Xaml.2.8.x64.appx -OutFile Microsoft.UI.Xaml.2.8.x64.appx;
+          Add-AppxPackage Microsoft.VCLibs.x64.14.00.Desktop.appx;
```

Comment:
Would these dependencies need to be updated here if the AppInstaller version changes, and the dependencies change?

Reply:
Yes, and once the Repair-WinGetPackageManager cmdlet is "fixed", I'll switch back to that method so we don't need to keep this updated.